ert3/evaluator/_evaluator.py

```diff
@@ -17,10 +17,6 @@
     EVTYPE_EE_TERMINATED,
 )
 
```

Comment:
Any reason for removing this?

Reply:
I don't see it actually. What was removed?